storage.go

```diff
@@ -24,21 +23,11 @@ func (s *Storage) create(path string, opt pairStorageCreate) (o *Object) {
 }
 
 func (s *Storage) createAppend(ctx context.Context, path string, opt pairStorageCreateAppend) (o *Object, err error) {
-	rp := s.getAbsPath(path)
-
-	options := make([]oss.Option, 0)
-	options = append(options, oss.ContentLength(0))
-
-	offset, err := s.bucket.AppendObject(rp, strings.NewReader(""), 0, options...)
-	if err != nil {
-		return
-	}
-
-	o = s.newObject(true)
+	o = s.newObject(false)
```

Comment:
We should create this object with `true`, as there is no more info we cloud provide.